opennlp-tools/src/main/java/opennlp/tools/ml/model/AbstractDataIndexer.java

```diff
@@ -188,17 +233,15 @@ protected List<ComparableEvent> index(ObjectStream<Event> events,
     return eventsToCompare;
   }
 
-  public int getNumEvents() {
-    return numEvents;
-  }
-
   /**
-   * Updates the set of predicated and counter with the specified event contexts and cutoff.
+   * Updates the set of predicates and counter with the specified event contexts and cutoff.
+   *
    * @param ec The contexts/features which occur in a event.
    * @param predicateSet The set of predicates which will be used for model building.
    * @param counter The predicate counters.
    * @param cutoff The cutoff which determines whether a predicate is included.
-   * @deprecated will be removed after 1.8.1 release
+   *
+   * @deprecated Use {{@link #update(String[], Map)}}. This method will be removed after 1.8.1 release
```

Comment:
"1.8.1" did happen a while ago? maybe time to remove? wdyt?

Reply:
@rzo1 It seems worth documenting the removal on its own, for clarity in release notes. I'll open a separate Jira for this.

Reply:
See [OPENNLP-1425](https://issues.apache.org/jira/browse/OPENNLP-1425).